SQL/2016-06-27-VideoUploader.sql

```diff
@@ -0,0 +1,52 @@
+
+-- Add 'videos' tab to the menu under Clinical section
+DELETE FROM LorisMenu WHERE Label='Videos';
+INSERT INTO LorisMenu (Label, Link, Parent, OrderNumber) VALUES ('Videos', 'videos/', (SELECT ID FROM LorisMenu as L WHERE Label='Clinical'), 5);
+
+-- Add 'videos' table
+DROP TABLE IF EXISTS `videos`;
```

Comment:
The patch shouldn't have a DROP table statement. There's a risk of it being accidentally run on existing projects.